src/main/java/com/example/Library/controller/BookController.java

```diff
@@ -0,0 +1,73 @@
+package com.example.Library.controller;
+
+import com.example.Library.model.Book;
+import com.example.Library.model.BookStats;
+import com.example.Library.service.BookService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+
+@RestController
+@RequiredArgsConstructor
+
+public class BookController {
+    private final BookService bookService;
+
+    @PostMapping("/books")
+    @ResponseStatus(HttpStatus.CREATED)
+    public Book addBook(@RequestBody Book book) {
+        return bookService.addBook(book);
+    }
+
+    @GetMapping("/books")
+    public List<Book> getBooks() {
+        return bookService.getBooks();
+    }
+
+    @GetMapping("/books/{bookID}")
+    @ResponseStatus(HttpStatus.NOT_FOUND)
+    public Book getBooksByID(@PathVariable Integer bookID) {
+        return bookService.getBookByID(bookID);
+    }
+
+    @PutMapping("/books/{bookID}")
+    @ResponseStatus(HttpStatus.NOT_FOUND)
+    public Book updateBook(@RequestBody Book newBookData, @PathVariable Integer bookID) {
+        return bookService.updateBook(newBookData, bookID);
+    }
+
+    @DeleteMapping("/books/{bookID}")
+    @ResponseStatus(HttpStatus.NOT_FOUND)
```

Comment:
W tym wypadku nawet jeśli znajdziemy książkę to dostaniemy 404 not found imo, źle